pkg/util/testutil/assert/assert.go

```diff
@@ -6,12 +6,13 @@ import (
 
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/go-cmp/cmp/cmpopts"
+	"google.golang.org/protobuf/testing/protocmp"
 )
 
 // Equals fails the test if `want` and `got` are not equal.
 func Equals(t *testing.T, want any, got any) {
 	t.Helper()
-	if diff := cmp.Diff(want, got, cmpopts.EquateErrors()); diff != "" {
+	if diff := cmp.Diff(want, got, cmpopts.EquateErrors(), protocmp.Transform()); diff != "" {
```

Comment:
The change to add protocmp.Transform() was necessary because we updated the CSI spec from v1.9.0 to v1.11.0, which changed how protobuf messages are structured internally.